schema.md

```diff
@@ -109,6 +109,29 @@
     \_ga=GA1.2.1076416180.1513383458; \_gid=GA1.2.1940452730.1513383458
     {"name": "example", "Browser": "Mozilla/5.0"}
     ```
+* __value_1000:__
+    * __Type:__ string
+    * __Description:__ The value that the function returned, restricted to 1000 characters.
+    * __Examples:__
+    ```
+    ""
+    {}
+    Mozilla/5.0 (X11; Linux x86_64; rv:52.0) Gecko/20100101 Firefox/52.0
+    \_ga=GA1.2.1076416180.1513383458; \_gid=GA1.2.1940452730.1513383458
+    {"name": "example", "Browser": "Mozilla/5.0"}
+    ```
+* __value_len:__
+    * __Type:__ Integer
+    * __Description:__ The length of the value a function returned
+    * __Examples:__ 59508
```

Comment:
Here and in the following entries, please remain consistent with the existing file and enclose the examples in a fenced code block, with multiple values on separate lines.